c/zosfile.c

```diff
@@ -763,6 +785,7 @@ int fileInfo(const char *filename, BPXYSTAT *stats, int *returnCode, int *reason
   reasonCodePtr = reasonCode;
 #endif
 
+  dumpbuffer(filename,strlen(filename));
```

Comment:
Please remove this naked `dumpbuffer` call.

Reply:
Fixed

Reply:
done!